app/Http/Requests/Jobs/UpdateRequest.php

```diff
@@ -26,12 +26,22 @@ public function authorize()
      */
     public function rules()
     {
-        return [
-            'name'        => 'required|max:60',
-            'price'       => 'required|numeric',
-            'worker_id'   => 'required|numeric',
-            'type_id'     => 'required|numeric',
-            'description' => 'max:255',
+        $rules = [
+            'name'      => 'required|max:60',
+            'price'     => 'required|numeric',
+            'worker_id' => 'required|numeric',
+            'type_id'   => 'required|numeric',
         ];
+
+        //Allow for flexibility instead of optionless hard-coded value for "description". This is
+        //achieved using environmental variable.
+        //A value of zero (0) will mean "no limit"
+
+        $char_len_job_description = intval(env('CHAR_LEN_JOB_DESCRIPTION', 255));
```

Comment:
Change this to **DESCRIPTION_CHAR_LEN**, too. And change variable name to **camelCase**.

Reply:
This line has not been fixed. Please use DESCRIPTION_CHAR_LEN and **camelCase** variable name